.github/workflows/ci.yml

```diff
@@ -0,0 +1,126 @@
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: [2.7, 3.6, 3.7, 3.8]
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v2
+              with:
+                python-version: ${{ matrix.python-version }}
+
+            - name: Install system dependencies
+              run: |
+                sudo apt-get update -y
+                sudo apt install libcurl4-openssl-dev libssl-dev
+                sudo apt-get install libgnutls28-dev
+
+            - name: Install Python dependencies
+              run: |
+                pip install --upgrade pip setuptools py
+                pip install -e .[all]
+
+    Black-Formatter:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Setup Python
+            uses: actions/setup-python@v2
+            with:
+              python-version: 3.8
+
+          - name: Run Black on Python
+            run: |
+              pip install black
+              ./run-tests.sh --check-black
+
+    Pytest:
+        runs-on: ubuntu-latest
+        strategy:
+          matrix:
+              python-version: [2.7, 3.6, 3.7, 3.8]
+
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Set up Python ${{ matrix.python-version }}
+            uses: actions/setup-python@v2
+            with:
+              python-version: ${{ matrix.python-version }}
+
+          - name: Install system dependencies
+            run: |
+              sudo apt-get update -y
+              sudo apt install libcurl4-openssl-dev libssl-dev
+              sudo apt-get install libgnutls28-dev
+
+          - name: Install Python dependencies
+            run: |
+              pip install --upgrade pip setuptools py
+              pip install -e .[all]
+
+          - name: Run pytest
+            run: ./run-tests.sh --pytest
+
+          - name: Codecov Coverage
+            if: matrix.python-version == 3.8
+            uses: codecov/codecov-action@v1
+            with:
+              token: ${{ secrets.CODECOV_TOKEN }}
+              files: coverage.xml
+
+    Docs-Build:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Setup Python
+            uses: actions/setup-python@v2
+            with:
+              python-version: 3.8
+
+          - name: Install system dependencies
+            run: |
+              sudo apt-get update -y
+              sudo apt install libcurl4-openssl-dev libssl-dev
+              sudo apt-get install libgnutls28-dev
+
+          - name: Install Python dependencies
+            run: |
+              pip install --upgrade pip setuptools py
+              pip install -e .[all]
+
+          - name: Install docs dependencies
+            run: pip install -e .[docs]
+
+          - name: Run Python manifest completeness
```

Comment:
For the GA CI actions list, please amend as we had discussed, so that all the checks would be visible on the main page directly, for example:

- lint/black
- lint/pydocstyle
- lint/flake8
- lint/check-manifest
- docs/sphinx
- tests/py27
- tests/py36
- tests/py37
- tests/py38
- build/docker

Reply:
Done 👍